README.md

```diff
@@ -36,6 +23,26 @@ This pipeline is doing XYZ.
     `$ pip install -r dev-requirements.txt`
     `$ vim local_config.py` # edit, edit
     `$ alembic upgrade head` # initialize database
+
+## Important note
+
+The pipeline will NOT send anything to SOLR/Metrics DB by default. You should trigger the update using a cronjob. There are two important modes:
```

Comment:
I don't understand the meaning behind the terms "normal operation" and "busy operation". Perhaps with some more explanation they will make more sense to me. At first glance the names don't directly convey information about how they are different. If they were named something like full record update and partial record update what they do would be more clear.

The readme refers to both normal operation and normal mode. If they are the same, we should only use one name.